http_concurrent_server.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import email.utils
+import os
+
+
+def server(socket, address):
+    buffersize = 32
+
+    try:
+        while True:
+
+            # conn, addr = server_socket.accept()
+            try:
+                while True:
```

Comment:
everything from this line up (except the buffer size assignment) should be removed.  Again, we fixed this today.  Where are those fixes?